.checkstyle/suppressions.xml

```diff
@@ -7,6 +7,7 @@
   <!-- no javadoc on test classes -->
   <suppress files="src[\\/]test[\\/]java[\\/].*" checks="FilteringWriteTag"/>
   <suppress files="src[\\/]test[\\/]java[\\/].*" checks="MissingJavadoc.*"/>
+  <suppress files="src[\\/]test[\\/]java[\\/].*" checks="JavadocPackage"/>
```

Comment:
a b c d e f g h i ***j*** k l ***m*** n o...

Reply:
what's an alphabet

Reply:
you tell me
![oh](https://avatars.githubusercontent.com/u/15833072?s=400&v=4)